README.md

```diff
@@ -75,15 +75,15 @@ Example deposit call
                     $currency,                                  /* Currency */
                     'SE',                                       /* Country */
                     NULL,                                       /* MobilePhone */
-                    NULL,                                       /* FirstName */
-                    NULL,                                       /* LastName */
+                    'Sam',                                      /* FirstName */
+                    'Trautman',                                 /* LastName */
                     NULL,                                       /* NationalIdentificationNumber */
-                    'Test',                                     /* ShopperStatement */
+                    NULL,                                       /* ShopperStatement */
                     $ip,                                        /* IP */
                     "$base_url/success.html",                   /* SuccessURL */
                     "$base_url/fail.html",                      /* FailURL */
                     NULL,                                       /* TemplateURL */
-                    "0",                                        /* URLTarget */
+                    NULL,                                       /* URLTarget */
```

Comment:
I Disagree, 0 is a perfectly valid URL Target.

Reply:
You are correct. I have updated the commit message to reflect the real problem.